extensions/cornerstone/src/commandsModule.ts

```diff
@@ -213,6 +218,8 @@ const commandsModule = ({ servicesManager }) => {
             CornerstoneViewportService,
           },
         });
+
+        return;
```

Comment:
This seems an odd thing to have a return statement here - it returns at the end of the method regardless, and there isn't more code after that.